gpu_monitor.py

```diff
@@ -183,9 +217,9 @@ def print_free_gpus(server, gpu_infos):
             info('\tGPU {}, {}'.format(gpu_info['idx'], gpu_info['model']))
 
 
-def print_gpu_infos(server, gpu_infos, run_ps, run_get_real_names,
-                    filter_by_user=None,
-                    translate_to_real_names=False):
+def print_gpu_infos(server, gpu_infos, run_ps, run_taskset,
```

Comment:
Instead of including the taskset functionality in `print_gpu_infos`, I would factor this out into its own method, e.g. `set_cpu_affinities`.